tests/provision/mock/fedora-rawhide-x86_64/multiple-tests/test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "run=\$(mktemp -d)" 0 "Create a run directory"
+        rlRun "pushd ../data"
+    rlPhaseEnd
+
+    rlPhaseStartTest
+        rlRun -s "TMT_SHOW_TRACEBACK=1 tmt --feeling-safe run --id $run -vvv"
+        rlAssertGrep "NAME.*Fedora Linux" $run/plan/execute/data/guest/*/test/os-release-*/output.txt
```

Comment:
It is fine to de-glob these. If ever these change, we will have an issue all across the tests, and at least this checks that there is no regression on that front.